TestFramework/ExtensionMethods.cs

```diff
@@ -273,5 +275,65 @@ public static IWebElement FindElementEnabledWithWait(IWebDriver webdriver, By fi
             webdriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(20);
             return null;
         }
+
+        public static bool IsElementEnabled(IWebDriver driver, By by)
+        {
+            try
+            {
+                driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(0);
```

Comment:
Can the Implicit wait be removed or replaced?

Reply:
@MichaelKeaneathmcts that can't go as the aim is not to wait until but rather check and return true or false if element is visible or not. If you don't override the implicit wait here, selenium would wait for whatever the default timeout is.

Reply:
OK for now.  But this would not be needed if there were no implicit waits at all.